lib/tmdb/client.rb

```diff
@@ -7,7 +7,7 @@ class Error < StandardError; end
     API_KEY = ENV['tmdb_api_key']
 
     class << self
-      def get_movie_search_results(movie_title)
+      def get_movie_title_search_results(movie_title)
```

Comment:
I renamed this from `get_movie_search_results` to `get_movie_title_search_results`  so it was more clearly different from `get_advanced_movie_search_results`.